HelpSource/static/scdoc.css

```diff
@@ -773,3 +773,80 @@ div.met_subclasses a.seemore {
 .subheader {
     color: var(--color-fg);
 }
+
+/* Copy button */
+.codeMirrorContainer {
+  position: relative;
+}
+
+.copy-button {
+  position: absolute;
+  top: 4px;
+  right: 10px;
+  background: none;
+  border: none;
+  cursor: pointer;
+  width: 20px;
+  height: 20px;
+  margin: 0;
+  padding: 0;
+  z-index: 1;
+
+  /* Hidden by default */
+  opacity: 0;
+  visibility: hidden;
+  transition: opacity 0.2s ease;
+}
+
+.codeMirrorContainer:hover .copy-button,
+.codeMirrorContainer:hover .copy-ico {
+  opacity: 1;
+  visibility: visible;
+}
+
+.codeMirrorContainer:not(:hover) .check-ico {
+  opacity: 0;
+  visibility: hidden;
+}
+
+.copy-ico,
+.check-ico {
+  width: 20px;
+  height: 20px;
+  display: block;
+  font-size: 18px;
+  color: var(--color-cm-comment);
```

Comment:
Not a request, just an idea/suggestion: `--color-cm-symbol` is green-ish in most themes and would maybe communicate better that the text was copied? Otherwise this is good as is.